assessment/schema/input-schema.json

```diff
@@ -0,0 +1,12 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "properties": {
+      "repositoryUrl": {
+        "type": "string"
+      }
```

Comment:
Add "pattern": "^(https?|git)(:\/\/|@)([^\/:]+)[\/:]([^\/:]+)\/(.+).git$" for validation